packages/airnode-deployer/src/cli/index.ts

```diff
@@ -91,14 +91,28 @@ yargs(hideBin(process.argv))
     }
   )
   .command(
-    'remove',
-    'Removes a deployed Airnode instance',
+    'remove-with-receipt',
+    'Removes a deployed Airnode instance using a receipt file',
     {
       receipt: {
         alias: 'r',
         description: 'Path to receipt file',
+        default: 'config/receipt.json',
         type: 'string',
       },
+    },
+    async (args) => {
+      logger.debugMode(args.debug as boolean);
+      logger.debug(`Running command ${args._[0]} with arguments ${longArguments(args)}`);
+
+      await runCommand(() => removeWithReceipt(args.receipt!));
+      return;
+    }
+  )
+  .command(
```

Comment:
I think that the implementation of this command can be done simpler. I had to make all the arguments optional when there was an option to use just the receipt but that's not the case anymore. All of them except the `project-id` which is GCP specific can be made [required](http://yargs.js.org/docs/#demandOption) and the code will become simpler.

Reply:
I've made the changes- significantly simpler!